File: core/src/main/kotlin/com/bing/lan/core/base/domain/EmailActive.java
package com.bing.lan.core.base.domain;

import org.apache.ibatis.type.Alias;

import java.util.Date;

import lombok.Getter;
import lombok.Setter;

/**
 * Created by 蓝兵 on 2018/5/11.
 */


@Alias("EmailActive")
@Getter
@Setter
public class EmailActive extends BaseDomain {

    public String uuidcode;
    public Long logininfoId;
    public String email;
    public Date sendDate;
}

File: core/src/main/kotlin/com/bing/lan/core/base/domain/EmailActive.kt
package com.bing.lan.core.base.domain

import lombok.Getter
import lombok.Setter
import org.apache.ibatis.type.Alias
import java.util.*

/**
 * Created by 蓝兵 on 2018/5/11.
 */


@Alias("EmailActive")
@Getter
@Setter
class EmailActive : BaseDomain() {

    public lateinit var uuidcode: String
    public var logininfoId: Long = 0
    public lateinit var email: String
    public lateinit var sendDate: Date
}
